cmd/skopeo/copy.go

```diff
@@ -41,17 +24,14 @@ func copyHandler(context *cli.Context) {
 	}
 	signBy := context.String("sign-by")
 
+	// TODO(runcom): unused digest??? Miloslav?
```

Comment:
(Strictly speaking it is printed to stdout, so not unused…)

The digest is there only because `dockerImage.retrieveRawManifest` was using it to eventually store it into a `DockerImageManifest`, and `skopeo inspect` is printing it to stdout. I did not look further into whether that should happen at all (though why print the unverified digest instead computing a fresh one, now that I look at it?) and simply preserved the original data flow.